lib/card_generator.rb

```diff
@@ -0,0 +1,35 @@
+class CardGenerator
+    attr_reader :text_file, :file_contents
+    def initialize(text_file)
+        @text_file = text_file
+        @file_contents = file_contents
+    end
+
+    def split_text
+        data_lines_array = @file_contents.split("\n")
+        data_lines_array.map! do |line|
+            line.split(",")
+        end
+    end
+
+    def open_file
+        @file_contents = File.read("./lib/"+@text_file)
+    end
+
+    def create_card(new_card, card_number, data_lines_array)
+        new_card = Card.new(data_lines_array[card_number][0].strip,data_lines_array[card_number][1].strip.downcase,data_lines_array[card_number][2].strip)
```

Comment:
This line is a little too long and each part could actually be placed in a variable to be more easier to read.